### ISSUE TITLE
Add NPC module

### ISSUE THREAD

**Issue by uiosun** [pbellchambers/rustyhack-mmo#186]
1. What can I do with game money?
2. Why we not has a NPC?

So, I think we can add NPC module to game, is right?

---

NPC is important, and I think so have some sub-feature:

1. Shoping, sell basic weapon / equipment / medicine / etc., buy player's material
2. Create safe area - walk around / attack monster in eyeshot / etc.
3. Told story - set a tutorial or FAQ / create atmosphere.

Has any idea?

**Comment by pbellchambers** [comment]
This is in the planned backlog: https://github.com/pbellchambers/rustyhack-mmo/projects?type=classic

No estimated date on when I can work on this though.

**Comment by uiosun** [comment]
Right, I will learn Rust and write simple code at second half year.

If you permit, I want take over this feature when the later time.
